bom/application/pom.xml

```diff
@@ -5619,18 +5619,6 @@
                     </exclusion>
                 </exclusions>
             </dependency>
-            <dependency>
-                <!-- this artifact has been relocated to com.mysql:mysql-connector-j -->
-                <groupId>mysql</groupId>
-                <artifactId>mysql-connector-java</artifactId>
```

Comment:
@gsmet could you please also add `mysql:mysql-connector-java` to https://github.com/quarkusio/quarkus/blob/main/independent-projects/enforcer-rules/src/main/resources/enforcer-rules/quarkus-banned-dependencies.xml